pkg/terraform/executor.go

```diff
@@ -2,59 +2,20 @@ package terraform
 
 import (
 	"bytes"
-	"os"
 	"os/exec"
 	"path/filepath"
-	"runtime"
-	"strings"
 
 	"github.com/openshift/installer/pkg/lineprinter"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
-// executor enables calling Terraform from Go, across platforms, with any
-// additional providers/provisioners that the currently executing binary
-// exposes.
-//
-// The Terraform binary is expected to be in the executing binary's folder, in
-// the current working directory or in the PATH.
-type executor struct {
-	binaryPath string
-}
-
-// Set the binary names for different platforms
-const (
-	tfBinUnix    = "terraform"
-	tfBinWindows = "terraform.exe"
-)
-
-// errBinaryNotFound denotes the fact that the Terraform binary could not be
-// found on disk.
-var errBinaryNotFound = errors.New(
-	"terraform not in executable's folder, cwd nor PATH",
-)
-
-// newExecutor initializes a new Executor.
-func newExecutor() (*executor, error) {
-	ex := new(executor)
-
-	// Find the Terraform binary.
-	binPath, err := tfBinaryPath()
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get Terraform binary's path")
-	}
-
-	ex.binaryPath = binPath
-	return ex, nil
-}
-
 // Execute runs the given command and arguments against Terraform.
 //
 // An error is returned if the Terraform binary could not be found, or if the
 // Terraform call itself failed, in which case, details can be found in the
 // output.
-func (ex *executor) execute(clusterDir string, args ...string) error {
+func Execute(clusterDir string, args ...string) error {
```

Comment:
No need to make this public.  Can we change this (and the name in the godoc above) to `execute`?